src/test/java/ay2122s1_cs2103t_w16_2/btbb/logic/commands/AddClientCommandIntegrationTest.java

```diff
@@ -43,7 +42,7 @@ public void execute_newClient_success() {
     public void execute_duplicateClient_throwsCommandException() {
         Client clientInList = model.getAddressBook().getClientList().get(0);
         ClientDescriptor clientInListDescriptor = new ClientDescriptorBuilder(clientInList).build();
-        assertCommandFailure(new AddClientCommand(clientInListDescriptor), model,
+        CommandTestUtil.assertCommandFailure(new AddClientCommand(clientInListDescriptor), model,
```

Comment:
Maybe import the assertCommandFailure method (like what's done for assertCommandSuccess),
then can change this to:
```suggestion
        assertCommandFailure(new AddClientCommand(clientInListDescriptor), model,
```